docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+```python exec
+import reflex as rx
+from pcweb.pages import docs
+```
+
+# Tutorial: Data Dashboard
+
+During this tutorial you will build a small data dashboard, where you can input data and it will be rendered in table and a graph. This tutorial does not assume any existing Reflex knowledge, but we do recommend checking out the quick [Basics Guide]({docs.getting_started.basics.path}) first. 
+
+The techniques you’ll learn in the tutorial are fundamental to building any Reflex app, and fully understanding it will give you a deep understanding of Reflex.
+
+
+This tutorial is divided into several sections:
+
+- Setup for the tutorial will give you a starting point to follow the tutorial.
+- Overview will teach you the fundamentals of Reflex: components and props.
+- Showing dynamic data (State) will show you how to use state to render data that will change in your app.
+- Using a Form to Add Data will show you how to let a user add data to your app and introduce event handlers.
+- Plotting Data in a Graph will show you how to use Reflex's graphing components.
+- Final Cleanup and Conclusion will explain how to further customize your app and add some extra styling to it.
+
+### What are you building?
+
+In this tutorial, you are building an interactive data dashboard with Reflex.
+
+You can see what the finished app and code will look like here:
+
+
+```python exec
```

Comment:
Maybe would be nice if the dashboard can be surrounded by some box or border. Just to clearly separate it out from the text in the page.

Reply:
fixed for all that needed it